Packs/SekoiaXDR/Integrations/SekoiaXDR/SekoiaXDR.py

```diff
@@ -0,0 +1,1482 @@
+    )
+
+    return CommandResults(
+        outputs_prefix="SekoiaXDR.ListAlerts",
+        outputs_key_field="short_id",
+        outputs=alerts["items"],
+    )
+
+
+def get_alert_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """Parameters"""
+    alert_uuid = args["id"]
+
+    alert = client.get_alert(alert_uuid=alert_uuid)
+    readable_output = tableToMarkdown(f"Alert {alert_uuid}:", alert)
+
+    return CommandResults(
+        readable_output=readable_output,
+        outputs_prefix="SekoiaXDR.Alert",
+        outputs_key_field="uuid",
+        outputs=alert,
+    )
+
+
+def query_events_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """Parameters"""
+    earliest_time = args["earliest_time"]
+    lastest_time = args["lastest_time"]
+    term = args["query"]
+    max_last_events = args.get("max_last_events")
+
+    jobQuery = client.query_events(
+        events_earliest_time=earliest_time,
+        events_latest_time=lastest_time,
+        events_term=term,
+        max_last_events=max_last_events,
+    )
+    readable_output = tableToMarkdown(
+        f"Event search created using the term {term}:", jobQuery
+    )
+
+    return CommandResults(
+        readable_output=readable_output,
+        outputs_prefix="SekoiaXDR.Events",
+        outputs_key_field="uuid",
+        outputs=jobQuery,
+    )
+
+
+def query_events_status_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """Parameters"""
+    search_job_uuid = args["uuid"]
+
+    status = client.query_events_status(event_search_job_uuid=search_job_uuid)
+    readable_output = tableToMarkdown(f"Status of the job {search_job_uuid}:", status)
+
+    return CommandResults(
+        readable_output=readable_output,
+        outputs_prefix="SekoiaXDR.Events",
+        outputs_key_field="search_job_uuid",
+        outputs=status,
+    )
+
+
+def retrieve_events_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """Parameters"""
+    search_job_uuid = args["uuid"]
+
+    events = client.retrieve_events(event_search_job_uuid=search_job_uuid)
+    readable_output = tableToMarkdown(
+        f"Events retrieved for the search {search_job_uuid}:", events
+    )
+
+    return CommandResults(
+        readable_output=readable_output,
+        outputs_prefix="SekoiaXDR.Events",
+        outputs_key_field="search_job_uuid",
+        outputs=events,
+    )
+
+
+def search_events_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """Parameters"""
+    earliest_time = args["earliest_time"]
+    lastest_time = args["lastest_time"]
+    term = args["query"]
+    max_last_events = args.get("max_last_events")
+    exclude_info_arg = args.get("exclude_info")
+    timeout_in_seconds = int(args["timeout_in_seconds"])
+    exclude_info_arg = args.get("exclude_info")
+    exclude_info_param = demisto.params().get("exclude_info_events")
+    replace_symbol = demisto.params().get("replace_dots_event")
```

Comment:
Hello @RotemAmit, I think for `get_remote_data_command` function those params are not needed. So there's no problem if they are None. ( For example `max_last_events ` is set to None because we need all alert events )